src/magnetVacuumFF.cpp

```diff
@@ -144,7 +144,7 @@ static Ref_t create_detector(Detector& det, xml_h e, SensitiveDetector /* sens *
     // -->the volume will be calculated at the end
     //-------------------------------------------
 
-    double endOfCentralBeamPipe_z = 445.580*dd4hep::cm; //extracted from central_beampipe.xml, line 64
+    double endOfCentralBeamPipe_z = 560.00*dd4hep::cm; // <-- extracted from central_beampipe.xml
```

Comment:
This should be fine. That's the one part of my magnet volume code which is done that way since we didn't have the numbers somewhere to be grabbed dynamically. @nat93 can you add the change to magnetVacuumFF.cpp to grab the number from line 147 from the central_beampipe.xml line 43?

Reply:
Hi @ajentsch and @wdconinc , thanks for your comments. Yes, I will fix it, following Alex's suggestion.